src/components/Buttons/Button/Button.scss

```diff
@@ -1,169 +1,213 @@
+                        &:hover,
+                        &:active {
+                            background-color: var(--color-background-#{$mappedContext}-bold-disabled);
+                        }
+                    }
+                }
+            } @else if $variant == outlined {
+                @if $mappedContext == neutral {
+                    color: var(--color-text-subtle);
+                    background-color: transparent;
+                    border-width: var(--line-normal);
+                    border-style: solid;
+                    border-color: var(--color-border-bold);
+    
+                  
+                    &[disabled],
+                    &.Button--isLoading {
+                        color: var(--color-text-subtlest);
+                        &:hover,
+                        &:active {
+                            background-color: var(--transparent);
+                        }
+                        border-color: var(--color-border-disabled);
+                    }
+
+                    
+                } @else {
+                    color: var(--color-text-#{$mappedContext}-default);
+                    background-color: transparent;
+                    border-width: var(--line-normal);
+                    border-style: solid;
+                    border-color: var(--color-border-#{$mappedContext}-bold-default);
+                    
+                    &[disabled],
+                    &.Button--isLoading {
+                        color: var(--color-text-#{$mappedContext}-disabled);
+                        &:hover,
+                        &:active {
+                            background-color: var(--transparent);
+                        }
+                        border-color: var(--color-border-#{$mappedContext}-subtle-brand);
+                    }
+
+                }
+            } @else if $variant == ghost {
+                @if $context == secondary {
+                    color: var(--color-text-subtle);
+                    background-color: var(--transparent);
+
+                    &:hover {
+                        background-color: var(--color-background-neutral-subtlest-hover);
+                    }
+
+                    &:active {
+                        background-color: var(--color-background-neutral-subtlest-pressed);
+                    }
+
+                    &:focus {
+                        box-shadow: 0px 0px 0px 2px var(--color-background-information-subtlest-pressed);
+                    }
+
+                    &[disabled],
+                    &.Button--isLoading {
+                        color: var(--color-text-disabled);
+                        background-color: var(--transparent);
+                        &:hover,
+                        &:active {
+                            background-color: var(--transparent);
+                        }
+                    }
+                } @else if $variant == ghost {
```

Comment:
we are already in the context of `if $variant == ghost` so it will always be `true`, so maybe we can just have `@else {...`.

Reply:
Good catch!